cmd/manager/main.go

```diff
@@ -168,6 +167,10 @@ func main() {
 		cmd.FatalError(setupLogger, err, "unable to create controller", "name", controllers.NodeLabelModuleVersionReconcilerName)
 	}
 
+	if err = controllers.NewMICReconciler(client, micAPI, mbscAPI, imagePullerAPI, scheme).SetupWithManager(mgr); err != nil {
```

Comment:
Why does the MIC controller needs the `mbscAPI` interface? I don't see it being used in the MIC controller.

Reply:
it reads the MBSC status